psm/src/Plutus/Model/Mock.hs

```diff
@@ -510,12 +510,12 @@ sendTx tx = do
 -}
 sendSingleTx :: Tx -> Run (Either FailReason Stat)
 sendSingleTx preTx@(Tx extra _) =
-  runValidate $
-    liftEither (processMints preTx) >>= \tx -> do
-      genParams <- gets (mockConfigProtocol . mockConfig)
-      case genParams of
-        AlonzoParams params -> checkSingleTx @Alonzo.Era params extra tx
-        BabbageParams params -> checkSingleTx @Babbage.Era params extra tx
+  runValidate $ do
```

Comment:
Just caught this while reading the module and it bugged me so I pulled the `>>= \tx ->` into the do block